src/commands/Minion/mine.ts

```diff
@@ -114,6 +129,16 @@ export default class extends BotCommand {
 				}
 			}
 		}
+		// Check if the user got a amulet of glory
+		if (ore.name.toLowerCase() === 'gem rock') {
+			for (const amulet of amulets) {
+				if (msg.author.hasItemEquippedAnywhere(amulet.id)) {
```

Comment:
probably do this in skilling since thats where you have to have prospectors equipped

Reply:
I just made it follow the same principle as the mining gloves so that ppl don't have to swap amulet of glory from melee/ranged/mage/skilling/misc

Reply:
ok, thats reasonable